ModelViewer/Data/Shaders/ads.frag

```diff
@@ -0,0 +1,36 @@
+varying lowp vec4 col;
```

Comment:
In order to compile on NVIDIA, the ADS shaders need to have #version 130 at the top.  This allows use of in/out keywords

Reply:
oh, okay. I'll make this change